packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -509,8 +509,15 @@ export class DashboardModel {
         dashboardUuid: string,
         dashboard: DashboardUnversionedFields,
     ): Promise<Dashboard> {
+        const withSpaceId = dashboard.spaceUuid
+            ? { space_id: await getSpaceId(this.database, dashboard.spaceUuid) }
+            : {};
         await this.database(DashboardsTableName)
-            .update(dashboard)
+            .update({
+                name: dashboard.name,
+                description: dashboard.description,
+                ...withSpaceId,
```

Comment:
Not sure why I can't use getSpaceId inline here 
![image](https://user-images.githubusercontent.com/1983672/180153225-a6a92fe5-c874-480f-bb19-78ee252caa67.png)